### PR TITLE
package names -> twitter_typesafe

### PR DESCRIPTION
**Pull request by rynonl**
To avoid namespace collisions

### CODE REVIEW

File: config/src/test/scala/com/typesafe/config/impl/ConcatenationTest.scala [hunk 1]
```diff
@@ -376,7 +376,7 @@ class ConcatenationTest extends TestUtils {
         assertEquals(Seq(1, 2), conf.getIntList("c.x").asScala)
     }
 
-    // from https://github.com/typesafehub/config/issues/177
+    // from https://github.com/config/issues/177
```
Comment (justinnguyen) [inline config/src/test/scala/com/typesafe/config/impl/ConcatenationTest.scala:379]: should leave the old link, no?

File: config/src/main/java/com/typesafe/config/impl/ConfigConditional.java [hunk 2]
```diff
@@ -1,10 +1,10 @@
-package com.typesafe.config.impl;
+package com.twitter_typesafe.config.impl;
 
-import com.typesafe.config.ConfigException;
+import com.twitter_typesafe.config.ConfigException;
 
 import java.util.Map;
 
-public class ConfigConditional {
+final class ConfigConditional {
```
Comment (justinnguyen) [inline config/src/main/java/com/typesafe/config/impl/ConfigConditional.java:7]: why this change?

**Comment by megankanne** [comment]
+1
